main.tf

```diff
@@ -79,21 +82,47 @@ resource "ibm_database" "postgresql_db" {
     }
   }
 
-  group {
-    group_id = "member" # Only member type is allowed for postgresql
-    memory {
-      allocation_mb = var.member_memory_mb
-    }
-    disk {
-      allocation_mb = var.member_disk_mb
-    }
-    cpu {
-      allocation_count = var.member_cpu_count
+  ## This for_each block is NOT a loop to attach to multiple group blocks.
+  ## This is used to conditionally add one, OR, the other group block depending on var.local.host_flavor_set
+  ## This block is for if host_flavor IS set
+  dynamic "group" {
```

Comment:
We should probably include this in a unit test. There are probably two different ways:
1. pick one of our examples, like "complete", and include a valid flavor (basic will have default, so both tests covered)
2. expose the new flavor in one of the examples, and have a new unit test in pr_test.go that would set the variable to a valid flavor (this would test the same example twice, one with and one without a flavor)

I think option 2 would work best.